libraries/helpers.rb

```diff
@@ -391,6 +392,25 @@ def pagespeed_url
       def default_site_template_source
         platform_family?('debian') ? "#{default_site_name}.conf.erb" : 'welcome.conf.erb'
       end
+
+      def apache_mod_php_package
+        case node['platform_family']
+        when 'debian'
+          "libapache2-mod-php#{node['php']['version'].to_f}"
+        when 'rhel', 'fedora'
+          'mod_php'
+        when 'suse'
+          "apache2-mod_php#{node['php']['version'].to_i}"
+        end
+      end
+
+      def apache_mod_php_filename
+        if platform_family?('debian')
+          "libphp#{node['php']['version'].to_f}.so" # libphp7.2.so
+        else
+          "libphp#{node['php']['version'].to_i}.so" # libphp7.so
+        end
+      end
```

Comment:
I don't think we want to depend on the php cookbook to figure this out. Instead, this helper method should determine what this will likely be based on the default repositories setup on the various platforms and the versions we support. Users can still override this via the resource using the ``module_name`` parameter if they use other repositories that may pull in different versions.

Reply:
Just to add to this we do not want to depend on any node attributes which are not automatic from ohai